### PR TITLE
[DOC] set up doc website with mkdoc

### PR DESCRIPTION
**Pull request by Remi-Gau**
- preview from my master branch: https://remi-gau.github.io/MOxUnit/

### CODE REVIEW

File: docs/acknowledgements.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Acknowledgements
+
+-   The object-oriented class structure was inspired by the [Python unit test] framework.
+-   The `assert*` function signatures are aimed to be compatible with Steve Eddin's [Matlab xUnit test framework].
+
+## Projects that use MOxUnit and MOcov
+
+-   [CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA)
+-   [bidspm](https://github.com/cpp-lln-lab/bidspm)
```
Comment (Remi-Gau) [inline docs/acknowledgements.md:9]: TODO
- add bids matlab

File: docs/acknowledgements.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
```
Comment (jorgepz) [inline docs/acknowledgements.md:10]:
```suggestion
-   [ONSAS](https://github.com/ONSAS/ONSAS): an Open Nonlinear Structural Analysis Solver.
```

File: run_tests.m
Comment (Remi-Gau) [inline run_tests.m:1]: - rename to make clear this moxunit in CI
- add a check of env variables to fail if not CI

File: docs/acknowledgements.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Acknowledgements
+
+-   The object-oriented class structure was inspired by the [Python unit test] framework.
+-   The `assert*` function signatures are aimed to be compatible with Steve Eddin's [Matlab xUnit test framework].
+
+## Projects that use MOxUnit and MOcov
+
+-   [CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA)
```
Comment (jorgepz) [inline docs/acknowledgements.md:8]:
```suggestion
-   [CoSMoMVPA](https://github.com/CoSMoMVPA/CoSMoMVPA): A multi-modal, multi-variate pattern analysis (MVPA) toolbox in Matlab / GNU Octave for cognitive neuroscientists.
```

File: run_tests_gh_ci.m [hunk 4]
```diff
@@ -0,0 +1,13 @@
+% Script used to run tests with github CI
+
+IS_CI = getenv('CI');
+if ~IS_CI
+    error('This script should only run in continuous integration.')
+end
```
Comment (Remi-Gau) [inline run_tests_gh_ci.m:6]: @nno 
renamed the script and added some 'child protection'

**Comment by Remi-Gau** [comment]
We are live: http://moxunit.github.io/MOxUnit/

**Pull request merged**